drivers/console/Kconfig

```diff
@@ -168,6 +168,52 @@ config IPM_CONSOLE_STACK_SIZE
 	  thread to print out incoming messages from the remote CPU. Specify the
 	  stack size for these threads here.
 
+config IPM_CONSOLE
+	bool "Inter-processor Mailbox console"
+	depends on IPM
+	select CONSOLE_HAS_DRIVER
+	help
+	  Enable console over Inter-processor Mailbox.
+
+config IPM_CONSOLE_ON_DEV_NAME
+	string "IPM device name used by console"
+	default "IPM_0"
```

Comment:
Could you add a default which takes the target device in DTS, similar to the UART console?

Reply:
I am thinking can I use something like: efaa9e311224f680f2900d5f3675104d6150e510. It is the same way IDC is used.

Reply:
That should work.

Reply:
Added getting target console device through DTS, it works, can be probably improved since I am not very fluent in Device tree yaml programming.